smarty/templates/email/account_details_changed.tpl

```diff
@@ -0,0 +1,14 @@
+Subject: Your Account Details Have Changed
+
+{$realname},
+
+Your account details have changed for the LORIS study {$study}.
+
+If you made these changes then you can ignore this email.
+
+If you did not make these changes, please contact your project administrator
+immediately.
```

Comment:
Hi {$realname},

Your account details have been changed for the LORIS study {$study}.

If you have made some changes then this email is a security notification.

If you did not make any changes, please contact your project administrator
immediately.